apps/gallery/js/gallery.js

```diff
@@ -646,10 +704,12 @@ function share(blobs, blobName) {
   // If all the blobs are image we use 'image/*'. If all are videos
   // we use 'video/*'. Otherwise, 'application/*'.
   var type;
-  if (types.length === 1 || types.every(function(t) { return t === types[0]; }))
+  if (types.length === 1 ||
+    types.every(function(t) { return t === types[0]; })) {
```

Comment:
nit: indent two spaces to align with top expression types.length === 1 for readibility